governance.md

```diff
@@ -0,0 +1,174 @@
+physical appearance, body size, race, age, religion, or national and ethnic origin.
+
+Individuals may apply for membership by submitting the required [membership
+form](https://docs.google.com/forms/d/e/1FAIpQLSdJbPczGHFN8mfMFu_YQym508OzFtOZxfSzr1sOoINxaMmiaw/viewform).
+Applications from people who currently have membership status revoked or applications that are found
+to be providing incorrect or fake information will be rejected.  Failure to abide by the Code of
+Conduct may result in the temporary suspension or permanent revocation of membership and access to
+US-RSE communication channels.
+
+Members are considered in “good standing” if they have supplied correct membership information
+during their application and are not subject to any form of sanction, suspension or disciplinary
+censure.
+
+### Voting
+
+When electing Steering Committee members, or when other membership elections take place, each US-RSE
+member in good standing will be entitled to one vote.
+
+### Steering Committee Roles and Number
+
+The Steering Committee consists of nine seats.
+
+The Steering Committee has the following roles:
+
+* The Steering Committee is responsible for setting the overall policy and direction of the US-RSE
+  Association, based on the inputs of the US-RSE Association membership.
+* The Steering Committee is ultimately responsible for day-to-day activities, operations, and
+  development, but many of these actions and activities will be initiated, organized, and managed by
+  community committees and members at large.
+* Members of the Steering Committee have the ability to grant or remove administrative rights for
+  US-RSE Association umbrella technologies including but not limited to:
+    * All repositories in the US-RSE GitHub organization
+    * All mailboxes and mailing lists associated with the us-rse.org domain name
+    * The us-rse.org domain and website
+    * The US-RSE slack channel
+    * Any other official accounts intended to represent the overall organization, now or in the
+      future (e.g., Twitter, Facebook, LinkedIn, etc.)
+
+The Steering Committee strives for consensus in its decisions. In the case of a dispute or argument
+within the community, a majority vote of all of the Steering Committee members will result in a
+final decision. In the event of a tie, the Steering Committee chair will cast the deciding vote.  At
+all Committee meetings, a quorum is at least one half of the members.
+
+### Steering Committee Elections and Terms
+
+After the conclusion of the founding Steering Committee's tenure, all future Steering Committee
+members will be elected by the general US-RSE membership.  Steering committee members are elected to
+a 2-year term. To provide continuity, steering committee members will have staggered terms, with
+four or five members elected each year.
+
+Nominations for open positions will be solicited from all members at least 6 weeks before the
```

Comment:
I think it is important that we make an active effort to ensure "all" members are aware of the nomination process and how to participate. I had suggested putting in some more process details here - such as "an email will be sent out to all members informing them of the nomination procedures." I am fine keeping the wording as-is, but I want to emphasize the importance of soliciting the nominations form "all" members!

Reply:
I agree with you, @jordanperr, we'll want to make lots of noise. I feel like we aren't a very "loud" organization, but leading up to the nomination/election we should be more persistent with emails, slack, etc.

Reply:
I agree we should make plenty of effort to notify everyone, but I don't think this document is the right place for specifics of how we do that.  I suggest we leave the wording as it is, but use this PR discussion as a note to make sure we have a good announcement system.

Reply:
I also don't think we need to say how in this document

Reply:
I agree to leave as is, esp. with the added clarification by Jordan in the announcement letter.